kuma/javascript/src/active-banner.jsx

```diff
@@ -216,6 +218,14 @@ function MDNBrowserCompatReportBanner() {
             url={
                 'https://mdn-web-dna.s3-us-west-2.amazonaws.com/MDN-Browser-Compatibility-Report-2020.pdf'
             }
+            onCTAClick={() => {
+                ga('send', {
+                    hitType: 'event',
+                    eventCategory: MDN_BROWSER_COMPAT_REPORT_ID,
+                    eventAction: 'Browser Compat Report CTA clicked',
```

Comment:
What is the context for how we've done this in the past? The reason I ask is; should we perhaps include **the year** in that action?

Reply:
We have not had this one in the past but, good idea 👍